README.md

```diff
@@ -2,5 +2,7 @@
 ## This is a sub-section
 This is some info....
 more info...
+even more info
```

Comment:
is this right¿ shouldnt be diferent, please change it to even more info with an exclamation mark